### PR TITLE
Bug/fix download csv method

### PR DESCRIPTION
**Pull request by BeccaSheldon**
@mrgordon hope the gemspec looks alright, let me know if you have suggestions! I'll be working on the ban and deban methods tomorrow. cc: @arvel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,36 @@
-# ruby-crowdflower
+CrowdFlower API Gem
+========
+Currently this is a toolkit for interacting with the CrowdFlower REST API. It may potentially become a complete Ruby gem for accessing and editing [CrowdFlower](http://crowdflower.com.com) jobs.
```
Comment (mrgordon) [inline README.md]: .com.com

**Comment by mrgordon** [comment]
Looks great! It might be helpful to put `gem install crowdflower` next to `require 'crowdflower'` so its clear that those two commands apply to people not using Bundler and the other two are for Bundler users.

File: README.md (hunk 1 above)
Comment (BeccaSheldon) [inline README.md]: ah sorry - the readme was messed up from the merge (I think) so I grabbed the one I had pushed, pre .com.com removal.

**Comment by BeccaSheldon** [comment]
Thanks @mrgordon! Just edited the readme Getting Started section to make installing and requiring a little clearer.

**Comment by mrgordon** [comment]
+1

**Comment by arvel** [comment]
+1

**Pull request merged**